plugins/plugin.xml

```diff
@@ -12,6 +12,7 @@
             </feature>
         </config-file>
         <source-file src="src/ios/OptimoveSDKPlugin.swift" />
+        <header-file src="src/ios/Bridging-Header-Header.h" type="BridgingHeader" />
```

Comment:
Think we've doubled up `-Header-Header` :)

Reply:
fixed